### PR TITLE
Add Code of Conduct link to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by JuanPabloDiaz**
PR created based on https://github.com/unitycatalog/unitycatalog-ui/pull/46#issuecomment-2262098041. Please review it @dennyglee 😎

### CODE REVIEW

**Review by Romanize — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ First off, thank you for considering contributing to unitycatalog-ui.
 
 We use [GitHub Issues](https://github.com/unitycatalog/unitycatalog-ui/issues) to track community-reported issues and [GitHub Pull Requests](https://github.com/unitycatalog/unitycatalog-ui/pulls) for accepting changes. If your contribution is not straightforward, please first discuss the change you wish to make by creating a new issue before making the change.
 
+## Code of Conduct
+
+Please review our [Code of Conduct](https://github.com/unitycatalog/unitycatalog/blob/main/CODE_OF_CONDUCT.md) before participating. We expect all contributors to adhere to the guidelines outlined there.
+
 ## Reporting issues
 
 Before reporting an issue on the
```
Comment (dennyglee) [inline CONTRIBUTING.md:13]: Thanks @JuanPabloDiaz  - the `Reporting Issues` seems incomplete.

**Review by dennyglee — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (JuanPabloDiaz) [reply CONTRIBUTING.md]: I am not sure what it is missing.

Comment (JuanPabloDiaz) [reply CONTRIBUTING.md]: done

**Review by dennyglee — APPROVED** [review]
LGTM!

**Comment by dennyglee** [comment]
Thanks very much @JuanPabloDiaz !

**Pull request merged**